cdap-ui/app/directives/widget-container/widget-complex-schema-editor/widget-complex-schema-editor.js

```diff
@@ -240,10 +240,8 @@ function ComplexSchemaEditorController($scope, EventPipe, $timeout, myAlertOnVal
   });
 
   $scope.$on('$destroy', () => {
-    EventPipe.cancelEvent('dataset.selected');
```

Comment:
Any specific reason why we are removing? We seem to be listening to it

Reply:
This shouldn't get removed, but we need to patch EventPipe to be able to remove a single callback function from an event. Currently `cancelEvent` will remove all callbacks